javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult =
+convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function name(Hey, Beavis) {
+  var name = 'Hey, Beavis';
+  return name;
+}
+
+var nameResult = name('Hey, Beavis');
+console.log('nameResult:', nameResult);
+
+function getArea(width, height) {
+  var multiply = width * height;
+  return multiply;
+}
+
+var getAreaResult =
+getArea(17, 42);
+console.log('getAreaResult:', getAreaResult);
+
+function getFirstName(person) {
+  var getFirstName = person;
+  return getFirstName;
+}
+
+var getFirstNameResult =
+getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' });
+console.log('getFirstNameResult:', getFirstNameResult);
```

Comment:
This `getFirstName` function needs to return `Lelouche` which is the `firstName` property of the object. Fix function so that it can also work with `getFirstName({ firstName: 'Ryan', lastName: 'Choi' })` and return `Ryan`.